wiki/Announcements/Contests/Monthly_Fanart_Contest/A_very_osu!_halloween/en.md

```diff
@@ -0,0 +1,95 @@
+# A very osu! Halloween
```

Comment:
Does osu-web shows the very first subtitle or the previous one? If the very first, you'll have to revise the title.